charts/stable/pihole/docs/split-dns.md

```diff
@@ -0,0 +1,55 @@
+# Automatic Split DNS with PiHole
+
+:::danger unrecommended configuration
+
+The below guide is not the recommended DNS solution for TrueCharts. We recommend running Blocky from the Enterprise train in a standalone configuration. Setting up your DNS as outlined below adds complexity and will reduce reliability.
```

Comment:
docusaurus will take the danger part as the 1 liner for doc overview, can you add before that danger tag so it displays a better simpified message?

_Using Blocky in conjunction with PiHole enables seamless automated split DNS functionality._ 

or something along those lines.